api/MessagePort.json

```
@@ -37,7 +37,7 @@
             "version_added": "5"
           },
           "safari_ios": {
-            "version_added": "5.1"
+            "version_added": "5"
```

You updated everything in MessageChannel.json. Should everything in this file also be 4.2?

```suggestion
            "version_added": "4.2"
```